src/samples/audio/smart_amp_test_llext/CMakeLists.txt

```diff
@@ -1,102 +1,51 @@
 # Copyright (c) 2023 Intel Corporation.
 # SPDX-License-Identifier: Apache-2.0
 
-# FIXME: This *WILL* be converted to add_llext_target() as long as that's
-# reasonably possible, and if it isn't, a *significant* effort will be made to
-# make that possible
-
-cmake_minimum_required(VERSION 3.20.0)
-find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project(smart_amp_test)
-
-SET_PROPERTY(GLOBAL PROPERTY TARGET_SUPPORTS_SHARED_LIBS TRUE)
+cmake_path(SET SOF_BASE NORMALIZE ${CMAKE_CURRENT_SOURCE_DIR}/../../../..)
```

Comment:
The quadruple `..` is quite ugly, how about `${APPLICATION_SOURCE_DIR}/..` instead?